frontend/src/components/MapView/MapTooltip/PointDataChart/PopupPointDataChart.tsx

```diff
@@ -51,16 +53,17 @@ const PopupPointDataChart = memo(() => {
     <div className={classes.chartContainer}>
       <div className={classes.chartSection}>
         <Chart
-          title={t(title)}
+          title={t(title, datasetParams)}
           config={config}
           data={dataset}
           xAxisLabel={
             isAdminBoundary(datasetParams)
               ? undefined
-              : t('Timestamps reflect local time in Cambodia')
+              : t('Timestamps reflect local time in region')
```

Comment:
Making this less custom. The translated values in Khmer still read Cambodia

Reply:
@gislawill - sorry, I didn't catch before that this change impacts the EWS-1294 flood layer as well. For that case, the previous text was more accurate. It illustrates to users that the timestamps are local to Cambodia. We're not adjusting based on users's location and it's not a generic region. This is how EWS-1294 reporting works